lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -422,6 +422,17 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'Path to <b>newrelic.yml</b>. If undefined, the agent checks the following directories (in order): <b>config/newrelic.yml</b>, <b>newrelic.yml</b>, <b>$HOME/.newrelic/newrelic.yml</b> and <b>$HOME/newrelic.yml</b>.'
         },
+        :'custom_events_instrumentation_topics' => {
+          :default => [],
+          :public => true,
+          :type => Array,
+          :allowed_from_server => true,
+          :dynamic_name => true,
+          :description => <<-DESCRIPTION
+A comma separated list of ActiveSupport custom event topics to subscribe to and
```

Comment:
I think the wording here could be changed since the type is set to Array. In yml files you can specify an array in a non comma separated list way. Like
```
custom_events_instrumentation_topics:
  - first_thing
  - second_thing
```
We could make it less specific and just say something like `an array of activesupport customer event topics`?

Reply:
@tannalynn do I then also need to include this disclaimer?

```yaml
  <Callout variant="caution">
    This option can't be set via environment variable.
  </Callout>
```

Reply:
hmm i thought that was only on our hash configs, but it looks like it is on the ignore/expected classes configs even though thats array. i don't think it says that on any other arrays? I looked at our env source class and it looks like it where does type stuff it doesn't have anything for arrays https://github.com/newrelic/newrelic-ruby-agent/blob/dev/lib/new_relic/agent/configuration/environment_source.rb#L82
So I guess I'm not sure! I always thought arrays could be passed in as a string on the env source and then parsed, but maybe that doesn't work at all.

Reply:
Gotcha. I've got to integration test the transaction name change anyhow, so I'll do some env var driven testing around this too. Thank you!

Reply:
Updated with https://github.com/newrelic/newrelic-ruby-agent/pull/1659/commits/951476b03bd2beafa3f9804a04b79b0b8a88ba6f